apps/server/src/modules/oauth/controller/api-test/oauth-sso.api.spec.ts

```diff
@@ -335,7 +335,7 @@ describe('OAuth SSO Controller (API)', () => {
 				const sourceSchool: School = schoolFactory.buildWithId(
 					{
 						systems: [sourceSystem],
-						officialSchoolNumber: '11111',
+						officialSchoolNumber: '11110',
```

Comment:
make this any differents?

Reply:
Unfortunately not. We will clean up the test data